src/vos/tests/vts_ts.c

```diff
@@ -474,6 +474,7 @@ lru_array_stress_test(void **state)
 		assert_int_equal(evicted, LRU_ARRAY_SIZE);
 	}
 
+	srand(1); /* The below test suite may fail for random seed value*/
```

Comment:
why is this necessary here?   Do we use random elsewhere in the code differently than before?

Reply:
This particular test  sometimes fails if the random number seed is different. During vos create the new code is calling uuid_create() which inturn is using rand. This inturn is causing this test suite to fail sometimes as the below for loop expects 70:30 proportion of lru_alloc vs lru_evict calls to be made and sometimes it is not exactly met.

Reply:
Ok, It's probably fine but I tried this several times with and without the envirable and it doesn't crash.